dxp/how_clay_arrives_to_dxp.md

```diff
@@ -117,61 +117,35 @@ buildCSS {
 
 ## How to import Clay into a custom-module?
 
-Before unreavel the final piece of this first part, we must understand what's a module and what's the difference with a theme.
+Before revealing the final piece of this first part, we must first understand what a module is, and what differentiates it from a theme
 
-In Liferay we refer to a module when we talk about an indipendent bundle that can be install in DXP. It usually contains configuration, markup and style. We often call them Widgets, Portlets or Themes.
+In Liferay we refer to a module when we talk about an independent bundle that can be installed in DXP. It usually contains configuration, markup and style. We often call them Widgets, Portlets or Themes.
 
-So as you probably already know by this moment, there is not (technical) difference between a custom-module and a custom-theme, and as for the custom-theme we can install Clay using
+So as you probably already know by this moment, there is no (technical) difference between a custom-module and a custom-theme, and as for the custom-theme we can install Clay using
 
-Node
-`npm install @clayui/css`
-
-Yarn
-`yarn add @clayui/css`
+| Node                      | Yarn                   |
+| ------------------------- | ---------------------- |
+| `npm install @clayui/css` | `yarn add @clayui/css` |
 
 [Read more at Clay's website](https://clayui.com/docs/get-started/how-to-use-clay.html)
 
 ## How to use Clay once imported?
 
 And finally we arrive at the section where we explain what to do with Clay once imported. In order to have Clay available in our CSS we'll need to import it in our `main.scss`, `style.scss` or `your-custom-name-css-file.scss`.
 
-It's a really simple code line:
+It's a really simple line of code:
 
 ```scss
 @import 'atlas-variables';
 ```
 
-Except, _it's not_ 😁.
-
----
-
-There are 4 different ways to import Clay CSS.
+Or if you are planning of use Clay Components CSS
```

Comment:
This is not even needed in DXP, only if you want to use clay components html. Since the theme already has Clay CSS, you don't need to add it again. That's why we recommend only using the variables.

Reply:
I think this is the main thing missing to round up this article. Check `top_head.jsp` to see how the different parts of a theme and clay are actually injected into the pages.